packages/e2e-test-app-fabric/test/RNTesterNavigation.ts

```diff
@@ -21,23 +22,30 @@ export async function goToComponentExample(example: string) {
  */
 export async function goToApiExample(example: string) {
   const componentsTabButton = await app.findElementByTestID('apis-tab');
+  const result = await componentsTabButton.waitForDisplayed({timeout: 20000});
+  // Work around for WebDriverIO inconsistent behavior.
+  if (!result) {
+    console.warn(
+      example,
+      'Page was skipped due to inconsistent WebDriverIO behavior. Please verify page can load locally or rerun this test.',
```

Comment:
Are there specific pages that are sometimes taking too long?

Reply:
This code can be removed now. Moving to running the automated tests in Release instead of Debug resolved this finnicky behavior.